### ISSUE TITLE
ARM64 support in Python module in oso-0.26.0

### ISSUE THREAD

**Issue by lambertjosh** [osohq/oso#1523]
It looks like the latest release of the python module (0.26.0) does not contain packages for ARM64. This can also be seen in the download files list on pypi: https://pypi.org/project/oso/0.26.0/#files. 

There were files for prior releases, such as 0.25.1: https://pypi.org/project/oso/0.25.1/

Unfortunately in comparing the tags [between 0.25.1 and 0.26.0](https://github.com/osohq/oso/compare/v0.25.1...v0.26.0) it is not clear to me why this happened, so unable to propose a PR.

**Comment by samscott89** [comment]
Hey @lambertjosh ! Thanks for raising this, looking into it now.

**Comment by samscott89** [comment]
You're absolutely right. The ARM wheels were built as part of the CI create release job (https://github.com/osohq/oso/actions/runs/1746903004) but for some reason not uploaded to pypi. I downloaded these and uploaded them manually, so now the ARM builds for 0.26 should be there.

I'll update this issue to reflect the underlying issue so we can track the problem here. Thank you again for letting us know there was a problem.

**Comment by samscott89** [comment]
I'm stumped. Leaving what I found.

If I download the `wheel.zip` artifact from the build, it has the ARM wheels. But the release job omitted it when zipping them and attaching to the tag

<details>
  <summary>Files uploaded to release</summary>
Run zip --junk-paths -r oso-python.zip wheel/
  adding: oso-0.26.0-pp37-pypy37_pp73-manylinux_2_12_x86_64.manylinux2010_x86_64.whl (deflated 1%)
  adding: oso-0.26.0-cp310-cp310-manylinux_2_12_x86_64.manylinux2010_x86_64.whl (deflated 1%)
  adding: oso-0.26.0-cp39-cp39-win_amd64.whl (deflated 1%)
  adding: oso-0.26.0-cp38-cp38-musllinux_1_1_x86_64.whl (deflated 1%)
  adding: oso-0.26.0-cp39-cp39-musllinux_1_1_x86_64.whl (deflated 1%)
  adding: oso-0.26.0-cp39-cp39-manylinux_2_12_x86_64.manylinux2010_x86_64.whl (deflated 1%)
  adding: oso-0.26.0-cp310-cp310-macosx_10_9_x86_64.whl (deflated 1%)
  adding: oso-0.26.0-cp37-cp37m-win_amd64.whl (deflated 1%)
  adding: oso-0.26.0-cp39-cp39-macosx_10_9_x86_64.whl (deflated 1%)
  adding: oso-0.26.0-cp38-cp38-win_amd64.whl (deflated 1%)
  adding: oso-0.26.0-cp36-cp36m-musllinux_1_1_x86_64.whl (deflated 1%)
  adding: oso-0.26.0-pp37-pypy37_pp73-macosx_10_9_x86_64.whl (deflated 1%)
  adding: oso-0.26.0-cp37-cp37m-macosx_10_9_x86_64.whl (deflated 1%)
  adding: oso-0.26.0-cp37-cp37m-manylinux_2_12_x86_64.manylinux2010_x86_64.whl (deflated 1%)
  adding: oso-0.26.0-pp37-pypy37_pp73-win_amd64.whl (deflated 1%)
  adding: oso-0.26.0-cp36-cp36m-win_amd64.whl (deflated 1%)
  adding: oso-0.26.0-cp310-cp310-musllinux_1_1_x86_64.whl (deflated 1%)
  adding: oso-0.26.0-cp36-cp36m-macosx_10_9_x86_64.whl (deflated 1%)
  adding: oso-0.26.0-cp310-cp310-win_amd64.whl (deflated 1%)
  adding: oso-0.26.0-cp38-cp38-manylinux_2_12_x86_64.manylinux2010_x86_64.whl (deflated 1%)
  adding: oso-0.26.0-cp37-cp37m-musllinux_1_1_x86_64.whl (deflated 1%)
  adding: oso-0.26.0-cp36-cp36m-manylinux_2_12_x86_64.manylinux2010_x86_64.whl (deflated 1%)
  adding: oso-0.26.0-cp38-cp38-macosx_10_9_x86_64.whl (deflated 1%)


</details>

No idea how the wheels are disappearing between the steps.

**Comment by lambertjosh** [comment]
Thanks for taking such a quick look @samscott89 and resolving the immediate issue with availability on PyPi.
